### PR TITLE
Fix some linting errors

### PR DESCRIPTION
**Pull request by ericvn**
Fix some linting errors which seem to be causing some PRs to not pass.

### CODE REVIEW

File: cni/cmd/istio-cni-repair/main.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func parseFlags() (filters *repair.Filters, options *ControllerOptions) {
 
 	pflag.Parse()
 	if err := viper.BindPFlags(pflag.CommandLine); err != nil {
-		log.Fatal("Error parsing command line args: %+v")
+		log.Fatal("Error parsing command line args: %+v", err)
```
Comment (howardjohn) [inline cni/cmd/istio-cni-repair/main.go:87]: should be fatalf

File: cni/cmd/istio-cni-taint/main.go [hunk 2]
```diff
@@ -156,7 +156,7 @@ func parseFlags() (options *ControllerOptions) {
 
 	pflag.Parse()
 	if err := viper.BindPFlags(pflag.CommandLine); err != nil {
-		log.Fatal("Error parsing command line args: %+v")
+		log.Fatal("Error parsing command line args: %+v", err)
```
Comment (howardjohn) [inline cni/cmd/istio-cni-taint/main.go:159]: same as above

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
p[ut a hold to fix the fatalf part, you can remove it yourself when fixed

File: cni/cmd/istio-cni-repair/main.go (hunk 1 above)
Comment (ericvn) [reply cni/cmd/istio-cni-repair/main.go]: updated.

File: cni/cmd/istio-cni-taint/main.go (hunk 2 above)
Comment (ericvn) [reply cni/cmd/istio-cni-taint/main.go]: updated?

**Pull request merged**